### PR TITLE
Update list of default surface reaction families and restrict Abstraction_Single_vdW

### PR DESCRIPTION
**Pull request by ChrisBNEU**
After discussions at the RMG-Cat hackathon, we decided to turn off the reaction families  for which we currently only have made up rules or a single training reactions in the `recommended.py` file by default. The reactions include `Surface_EleyRideal_Addition_Multiple_Bond`, `Surface_Migration`, `Surface_DoubleBond_to_Bidentate`, and `Surface_Addition_Single_vdW`. We anticipate that the elementary steps discovered by these families will cause problems in the microkinetic models due to the unreasonable rate estimates. 

Additionally, we restricted the `Surface_Abstraction_Single_vdW` to not abstract other atoms with an adsorbed H and removed physisorbed H2 from the tree (see discussion  [#2494](https://github.com/ReactionMechanismGenerator/RMG-Py/discussions/2495)) 

An example of the reactions that we now prevent is shown here:
image
In this case, XH abstracts an OH group from physisorbed CH3OH, which is unreasonable.  

We updated the top level BEP rule of `Surface_Abstraction_Single_vdW` with the BEP from `Surface_Abstraction_Beta_vdW` to provide better rate estimates.

### CODE REVIEW

File: input/kinetics/families/Surface_Abstraction_Single_vdW/groups.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 """
 multiplicity [1]
 1 *1 Xv u0 p0 c0
-2 *2 R  u0 px cx {3,S}
+2 *2 R!H  u0 px cx {3,S}
```
Comment (bjkreitz) [inline input/kinetics/families/Surface_Abstraction_Single_vdW/groups.py:38]: Atom 2 should also not be an H. With this fix, the family works

**Review by bjkreitz — COMMENTED** [review]
I think the reaction family makes now more sense. However, I think we should make it more restrictive. I don't think that C-C bonds should be broken from vdW species, as this is should be energetically unfeasible. 

We should change the rule. Currently, this is made up. We could use the general BEP relations from the Surface_Abstraction_Beta_vdW family to at least give it some reasonable values.

File: input/kinetics/families/Surface_Abstraction_Single_vdW/groups.py (hunk 1 above)
Comment (ChrisBNEU) [reply input/kinetics/families/Surface_Abstraction_Single_vdW/groups.py:38]: could we use this PR to change Surface_Abstraction_vdW and surface_abstraction_single_vdw to have this for a top level node (basically stealing from the abstraction beta top node) : 
```
entry(
    index = 1,
    label = "Abstracting;Donating",
    kinetics = SurfaceArrheniusBEP(
        A = (4.18e17, 'm^2/(mol*s)'),
        n = 0.,
        alpha =0.94,
        E0 = (129.3, 'kJ/mol'),
        Tmin = (200, 'K'),
        Tmax = (3000, 'K'),
    ),
    rank = 0,
    shortDesc = u"""Default""",
    longDesc = u""" 
These numbers for the general BEP are from the abstraction reaction of C-H to O.
BEP values from "Combined DFT, Microkinetic, and Experimental Study of Ethanol Steam Reforming on Pt", Sutton et al., The Journal of Physical Chemistry C, 2013, 117, 4691-4706, DOI:10.1021/jp312593u
From Table 7 includes beta and alpha position. Pre-exponential coefficient is calculated from 1e13 s^-1 (standard guess from transition state theory) divided by 2.39e-9 mol cm^-2 (surface site density of Pt(111)
"""
)
```

**Comment by bjkreitz** [comment]
In this PR, we moved a couple of surface families for which we have only made up rules or just one training reaction into the surface development section of the `recommended.py` file. These families shouldn't be used by default.  
The following reactions were moved: `Surface_EleyRideal_Addition_Multiple_Bond`, `Surface_Migration`, `Surface_DoubleBond_to_Bidentate`, `Surface_Addition_Single_vdW`.

**Comment by bjkreitz** [comment]
I accidentally force-pushed a commit without considering Richard's rebase. I think another rebase should fix this issue.

**Comment by ChrisBNEU** [comment]
The superminimal test got the error below, so it was unable to execute. I think it's unrelated to the changes, but I also don't know why it failed. 
```
2024-03-20T14:50:17.7661909Z 1 radicals on CH3CH2OO exceeds limit of 0. Using HBI method.
2024-03-20T14:50:17.7973897Z Trying MopacMolPM3 attempt 1 of 10 on molecule CCOO.
2024-03-20T14:50:18.1042407Z Warning: Mopac error message:
2024-03-20T14:50:18.1317992Z Traceback (most recent call last):
2024-03-20T14:50:18.1319874Z   File "/usr/share/miniconda3/envs/rmg_env/lib/python3.7/site-packages/julia/pseudo_python_cli.py", line 308, in main
2024-03-20T14:50:18.1321379Z     python(**vars(ns))
2024-03-20T14:50:18.1323040Z   File "/usr/share/miniconda3/envs/rmg_env/lib/python3.7/site-packages/julia/pseudo_python_cli.py", line 59, in python
2024-03-20T14:50:18.1324601Z     scope = runpy.run_path(script, run_name="__main__")
2024-03-20T14:50:18.1325816Z   File "/usr/share/miniconda3/envs/rmg_env/lib/python3.7/runpy.py", line 263, in run_path
2024-03-20T14:50:18.1327002Z     pkg_name=pkg_name, script_name=fname)
2024-03-20T14:50:18.1328192Z   File "/usr/share/miniconda3/envs/rmg_env/lib/python3.7/runpy.py", line 96, in _run_module_code
2024-03-20T14:50:18.1329443Z     mod_name, mod_spec, pkg_name, script_name)
2024-03-20T14:50:18.1330592Z   File "/usr/share/miniconda3/envs/rmg_env/lib/python3.7/runpy.py", line 85, in _run_code
2024-03-20T14:50:18.1332048Z     exec(code, run_globals)
2024-03-20T14:50:18.1332630Z   File "rmg.py", line 118, in <module>
2024-03-20T14:50:18.1333186Z     main()
2024-03-20T14:50:18.1333548Z   File "rmg.py", line 112, in main
2024-03-20T14:50:18.1334046Z     rmg.execute(**kwargs)
2024-03-20T14:50:18.1335080Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/rmg/main.py", line 752, in execute
2024-03-20T14:50:18.1336084Z     self.initialize(**kwargs)
2024-03-20T14:50:18.1337163Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/rmg/main.py", line 550, in initialize
2024-03-20T14:50:18.1338178Z     self.load_database()
2024-03-20T14:50:18.1339248Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/rmg/main.py", line 471, in load_database
2024-03-20T14:50:18.1340514Z     family.add_rules_from_training(thermo_database=self.database.thermo)
2024-03-20T14:50:18.1342139Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/data/kinetics/family.py", line 1134, in add_rules_from_training
2024-03-20T14:50:18.1343865Z     reactant.thermo = thermo_database.get_thermo_data(reactant, training_set=True, metal_to_scale_to=metal)
2024-03-20T14:50:18.1347103Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/data/thermo.py", line 1329, in get_thermo_data
2024-03-20T14:50:18.1348239Z     tdata = self.estimate_radical_thermo_via_hbi(molecule, quantum_mechanics.get_thermo_data)
2024-03-20T14:50:18.1349266Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/data/thermo.py", line 2049, in estimate_radical_thermo_via_hbi
2024-03-20T14:50:18.1350299Z     thermo_data_sat = stable_thermo_estimator(saturated_struct)
2024-03-20T14:50:18.1351211Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/main.py", line 216, in get_thermo_data
2024-03-20T14:50:18.1351895Z     thermo0 = qm_molecule_calculator.generate_thermo_data()
2024-03-20T14:50:18.1352691Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/molecule.py", line 451, in generate_thermo_data
2024-03-20T14:50:18.1353362Z     self.qm_data = self.generate_qm_data()
2024-03-20T14:50:18.1354054Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/mopac.py", line 339, in generate_qm_data
2024-03-20T14:50:18.1354652Z     success = self.run()
2024-03-20T14:50:18.1355215Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/mopac.py", line 117, in run
2024-03-20T14:50:18.1355775Z     return self.verify_output_file()
2024-03-20T14:50:18.1356462Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/mopac.py", line 203, in verify_output_file
2024-03-20T14:50:18.1357080Z     qm_data = self.parse()
2024-03-20T14:50:18.1358038Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/molecule.py", line 425, in parse
2024-03-20T14:50:18.1358626Z     cclib_data, radical_number + 1
2024-03-20T14:50:18.1359291Z   File "/home/runner/work/RMG-database/RMG-database/RMG-Py/rmgpy/qm/qmdata.py", line 101, in parse_cclib_data
2024-03-20T14:50:18.1360038Z     rotational_constants = (cclib_data.rotcons[-1], 'cm^-1')
2024-03-20T14:50:18.1360443Z IndexError: list index out of range
2024-03-20T14:50:20.6667733Z superminimal Failed to Execute
```

**Review by ChrisBNEU — COMMENTED** [review]
I can't approve of this pr because I was the one who wrote it, however I think it is ready to merge. It passed the unit tests, and fails the superminimal regression tests because of the issue addressed in #2535 (cclib v1.8).

**Review by bjkreitz — APPROVED** [review]
Everything looks fine now. I restarted the unit tests and the superminimal example passed.

**Comment by ChrisBNEU** [comment]
I am not sure why the previous regression tests, but it looks like a julia dependency issue and not anything to do with the changes here. We are good to merge.

**Comment by ChrisBNEU** [comment]
@rwest if you have a minute this pr is ready.

**Pull request merged**